Setting Up Cron & php7.3-intl

coldestheart opened the pull request:
Setting up **Cron** for Grav in container; Add **php7.3-intl** for **twig-extentions** plugin.

Commit:
Setting Up Cron & php7.3-intl

Commit:
Setting Up Cron & php7.3-intl & fix RUN

dsavell commented on Dockerfile.gravcore:
```diff
@@ -1,4 +1,4 @@
-FROM bitnami/minideb:stretch
+FROM bitnami/minideb:latest
```
I do not believe in referencing latest for OS baselines, due to not knowing changes between operating system versions, would like to keep this controlled.

coldestheart replied:
I understand, but with Buster works perfect

dsavell replied:
More than happy to set it to buster.

dsavell commented on Dockerfile.gravcoreadmin:
```diff
@@ -1,4 +1,4 @@
-FROM bitnami/minideb:stretch
+FROM bitnami/minideb:latest
```
Same as above

dsavell commented:
Thanks for the contribution, Reviewing

Commit:
Change OS baseline

Commit:
Change OS baseline

dsavell commented on root/init-core:
```diff
@@ -54,4 +54,8 @@ umask 0002
 
 service php7.3-fpm start
 service nginx start
+## start cron
```
This will probably be needed in root/init-admin also.

coldestheart replied:
Thx for review )

coldestheart committed:
Update cron startup && exit trap

coldestheart committed:
Enable nginx cahing

Good optimization for online speed performance testers (like Google pagespeed, gtmetrix and other), and not only

coldestheart committed:
Update default
